Frontend/ui-driver/src/Screens/ReferralScreen/View.purs

```diff
@@ -467,6 +467,7 @@ rankCard item aboveThreshold state =
           , height WRAP_CONTENT
           , gravity CENTER_HORIZONTAL
           , margin (MarginHorizontal 8 6)
+          , visibility $ if aboveThreshold && (item == currentDriverData && currentDriverData.rank == 0) then GONE else VISIBLE
```

Comment:
can we put this `aboveThreshold && (item == currentDriverData && currentDriverData.rank == 0)` inside a function as its being repeated

Reply:
added the function